### PR TITLE
button to go to page 2 works

### PR DESCRIPTION
**Pull request by taks-mohs**
I added the functionality to the button to go to page 2.

### CODE REVIEW

File: screens/FirstPage.js [hunk 1]
```diff
@@ -2,18 +2,18 @@ import { View, StyleSheet } from "react-native";
 
 import MainButton from "../components/MainButton";
 
-function MainPage() {
+function FirstPage(props) {
     return (
         <View style={styles.container}>
             <MainButton>Main Page</MainButton>
-            <MainButton>Second Page</MainButton>
+            <MainButton whenPressed={props.changePageHandler} >Second Page</MainButton>
```
Comment (seanstaka) [inline screens/FirstPage.js:9]: You should also make sure you can get back to the main page.

**Review by seanstaka — CHANGES_REQUESTED** [review]
You should add a button on page to that will let you go back to the main page

**Comment by taks-mohs** [comment]
@seanstaka It can go from page 2 to the main now.  Can you check it out?

**Review by seanstaka — APPROVED** [review]
Looks good!

**Pull request merged**